brightwind/demo_datasets/solar_iea43_wra_data_model_v1_3.json

```diff
@@ -0,0 +1,383 @@
+{
+  "author": "Brighthub",
+  "organisation": "Brightwind",
+  "date": "2025-03-19",
+  "version": "1.3.0-2024.03",
+  "measurement_location": [
+    {
+      "uuid": "1be02b4a-test",
```

Comment:
@olivia-bentley can we use a correctly formatted uuid here as it doesn't validate against the data model schema?
@BiancaMorandi